examples/basic_voice.py

```diff
@@ -1,30 +1,28 @@
 import asyncio
```

Comment:
As a library we probably shouldn't be supporting 'official' voice examples using youtube given discords current stance.